core/test/solver/ir.cpp

```diff
@@ -148,7 +148,7 @@ TEST_F(Ir, CanBeCleared)
 
 TEST_F(Ir, CanSetInnerSolver)
 {
-    auto cg_factory =
+    auto ir_factory =
```

Comment:
I think technically, it was supposed to be a CG solver. For the purpose of the test,maybe it is okay, but for clarity, I would prefer if you could create an alias for CG and use the CG solver as an inner solver here, because that is what this is meant to test.

Reply:
Oh, you mean that the iterative solver was acting as an CG solver?
I just saw that every solver has the exact same pattern for this test, and it looked like a copy-paste error to me. And since CG is neither included, nor mentioned anywhere else in the test (`Solver` is an alias to `gko::solver::Ir<>`), I thought it was just a typo.

What alias do you want me to add?

Reply:
Yes, I meant something like `using CGSolver=gko::solver::Cg<>` and using this in the inner solver factory.